mne/epochs.py

```diff
@@ -809,14 +809,20 @@ def __next__(self, *args, **kwargs):
         """Provide a wrapper for Py3k."""
         return self.next(*args, **kwargs)
 
-    def average(self, picks=None):
-        """Compute average of epochs.
+    def average(self, picks=None, method="mean"):
+        """Compute an aggregate over epochs, by default the mean.
 
         Parameters
         ----------
         picks : array-like of int | None
             If None only MEG, EEG, SEEG, ECoG, and fNIRS channels are kept
             otherwise the channels indices in picks are kept.
+        method : str | callable
+            How to combine the data. If "mean", "median" or "std", the mean,
```

Comment:
@jona-sassenhagen ok with the idea but then don't avertise std and say that it's meant to do some kind of averaging as this will get stored in a fif file as an average

Reply:
Yes agreed std should not be in the list